### PR TITLE
Remove BatchDocumentUpdateMessage and BatchDocumentUpdateReply

### PR DESCRIPTION
**Pull request by toregge**
from java portion of documentapi.

@vekterli: please review
@geirst, @gjoranv: FYI

### CODE REVIEW

**Review by vekterli — APPROVED** [review]
:+1:

File: documentapi/src/main/java/com/yahoo/documentapi/messagebus/protocol/DocumentProtocol.java [hunk 1]
```diff
@@ -66,7 +66,6 @@
     public static final int MESSAGE_EMPTYBUCKETS = DOCUMENT_MESSAGE + 23;
     public static final int MESSAGE_REMOVELOCATION = DOCUMENT_MESSAGE + 24;
     public static final int MESSAGE_QUERYRESULT = DOCUMENT_MESSAGE + 25;
-    public static final int MESSAGE_BATCHDOCUMENTUPDATE = DOCUMENT_MESSAGE + 26;
```
Comment (gjoranv) [inline documentapi/src/main/java/com/yahoo/documentapi/messagebus/protocol/DocumentProtocol.java]: This package is marked as PublicApi in package-info.java, so public fields cannot be removed from the master branch, unless you make sure nobody is using them.

File: documentapi/src/main/java/com/yahoo/documentapi/messagebus/protocol/RoutableFactories50.java [hunk 2]
```diff
@@ -583,86 +583,6 @@ protected boolean doEncode(DocumentReply obj, DocumentSerializer buf) {
         }
     }
 
-    public static class BatchDocumentUpdateMessageFactory extends DocumentMessageFactory {
```
Comment (gjoranv) [inline documentapi/src/main/java/com/yahoo/documentapi/messagebus/protocol/RoutableFactories50.java]: Public class in PublicApi package. Same goes here. You have to make sure nobody is using it before removing, or remove it on '7'.

**Review by gjoranv — COMMENTED** [review]
If you verify that no hosted apps or other known customers (check git.ouroath) use the commented apis, go ahead. Otherwise fix those parts on 7 instead.

**Pull request merged**